### DISCUSSION TITLE
Integratin with Jira Work Management

### DISCUSSION THREAD

**Discussion by julia-aek** [community/community#67469] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Projects

### Body

There are a lot of manuals how to integrate Jira Software with GitHub. But I can't find how to integrate Jira Work Management with GitHub. Is it possible? If it is, could you give me some gides?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @julia-aek ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
